scripts/travis/build-all-in-one-image.sh

```diff
@@ -13,7 +13,11 @@ set -x
 make build-all-in-one GOOS=linux
 
 export REPO=jaegertracing/all-in-one
-docker build -f cmd/all-in-one/Dockerfile -t $REPO:latest cmd/all-in-one
+if [ "$(uname -m)" = "x86_64" ]; then
```

Comment:
why is this change only for all-in-one and not all other Docker images?

Reply:
Well...this is a little bit awkward because the all-in-one image is the only one I need for now...

But I would be more than happy to support all of them if you need :P

Reply:
Doing this half-way means someone else will get burned later on. If we support arm64, we should do it for all binaries in all distribution forms.

Reply:
I did some tests on supporting other distribution forms and didn't see any big problems for other binaries and images. The only issue is related to your last comment which is related to file names.

Reply:
Resolved this comment as currently this PR support all images